pkg/build/nodeimage/build_impl.go

```diff
@@ -97,6 +97,9 @@ func (c *buildContext) buildImage(bits kube.Bits) error {
 		if err := execInBuild("chmod", "+x", nodePath); err != nil {
 			return err
 		}
+		if err := execInBuild("chown", "root:root", nodePath); err != nil {
```

Comment:
do we need to do this recursive? `chown -R root:root`

Reply:
no, this is a full filepath.

Reply:
ahh, nodePath points to binary, scratch thath